.env.example

```
@@ -1,4 +1,5 @@
 PUBSUB_EMULATOR_HOST=localhost:3004
+AIVEN_TOKEN=very-aiven-token
```

Burde enten slettes
```suggestion
```

Eller ha korrekt default verdi
```suggestion
AIVEN_TOKEN=fake-aiven-token
```